application/models/Fleet.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * Dummy model class that represent our airline's fleet
+ */
+class Fleet extends CI_Model
+{
+
+    // The mock plane data
+	var $data = array(
```

Comment:
There's some inconsistent tabs spacing in this whole file. Please double check. Yes I know this is very picky.

Reply:
i admire your attention to detail.
shoudl we go with 4 space indents everywhere?

Reply:
Ok let's go with 4 space indents.